src/docs/asciidoc/15-examples.adoc

```diff
@@ -30,6 +30,52 @@ task buildImage(type: DockerBuildImage) {
 }
 ----
 
+==== Interacting With a Container Running in the Background
+
+This code demonstrates managing a container running in the background with this plugin
+which can facilitate locally running a Dockerized application. This can be done by
+creating the container with an explicit `containerName` which is then used as
+the `targetContainerId` for relevant tasks.
+
+.build.gradle
+[source,groovy]
+----
+apply plugin: 'com.bmuschko.docker-remote-api'
+
+import com.bmuschko.gradle.docker.tasks.container.*
+
+ext {
+  CONTAINER_NAME = 'myapp'
+} 
+
+task buildMyAppImage(type: DockerBuildImage) {
+    inputDir = file('docker/myapp')
+    tag = 'test/myapp:latest'
+}
+
+task dockerCreateContainer(type: DockerCreateContainer) {
+    dependsOn buildMyAppImage
+    targetImageId { buildMyAppImage.imageId }
+    containerName = CONTAINER_NAME
```

Comment:
Can you explain why using the container name is used (maybe even preferred) over the container ID in this example? Sure, you can use either or but why would a user want to go for the container name instead?

Reply:
sure, did you want that as a comment here or included in the docs?

The reason is that the containerId will be unique per created container and therefore would need to be tracked when interacting with a container that is started in the background (i.e. the containerId would normally be available over the course of a single invocation of gradle and therefore not usable in a subsequent invocation to do something like stop a running container). Additionally in this use case the containers are ephemeral and therefore the identifier is likely reusable. So: the idea of eagerly feeding in a defined identifier rather than a generated unique identifier allows for a simple reliable handle for the resulting container.